### PR TITLE
My plan: update feature cards

### LINKED ISSUE #10582: Improve Jetpack plans page in wp-admin
The Plans page in wp-admin > Jetpack Dashboard > Plans is very long and visually clunky. Screenshot: https://cloudup.com/ccKVlFXdwyJ

Let's update this view to reflect the changes we've made on the Calypso side in Plan > My Plan (with the feature cards) so items are easier to read/understand and are more actionable for users.

### PR DESCRIPTION
**Pull request by keoshi**
Fixes #10582

#### Changes proposed in this Pull Request:

* Updates plan feature cards content.
* Updates plan feature cards design.

#### Before

image

#### After

image

#### Testing instructions:

* Fire up this branch on your site.
* To to `/wp-admin/admin.php?page=jetpack#/my-plan`.
* Make sure you see the feature cards with updated content and styles.

#### Proposed changelog entry for your changes:

* My plan: update content and design of plan feature cards.

### CODE REVIEW

**Review by MichaelArestad — COMMENTED** [review]
Right now on the free plan, there is not an even number of cards so the last card looks a bit funky:

image

I suggested making the last block full width if it's an odd number. The easiest way to do that is to switch to flexbox like this:

https://codepen.io/MichaelArestad/pen/5812bffebabf909bd3148a3c81eeae04/?editors=1100

**Comment by keoshi** [comment]
Thanks for catching that, @MichaelArestad ! Unfortunately, that won't work in this case but I've added `&:nth-child(2n + 1):nth-last-child(1) { min-width: 100% }` which should have the same effect.

File: _inc/client/my-plan/my-plan-body.jsx [hunk 1]
```diff
@@ -286,11 +286,11 @@ class MyPlanBody extends React.Component {
 						( 'inactive' !== this.props.getModuleOverride( 'videopress' ) ) && (
 							<div className="jp-landing__plan-features-card">
 								<h3 className="jp-landing__plan-features-title">{ __( 'Video Hosting' ) }</h3>
-								<p>{ __( 'Fast, optimized, ad-free, and unlimited video hosting for your site.' ) }</p>
+								<p>{ __( 'High-speed, high-definition video hosting that uses your server space efficiently, and comes with no third-party ads.' ) }</p>
```
Comment (jeherve) [inline _inc/client/my-plan/my-plan-body.jsx]: I am not sure I understand "uses your server space efficiently". VideoPress videos are not uploaded at all on the site owner's server, so it does not use the server at all.

**Review by jeherve — CHANGES_REQUESTED** [review]
I would just have one small remark about wording. Overall it works well.

Worth noting that I still see the odd number of cards:

screenshot 2018-12-13 at 14 04 29

File: _inc/client/my-plan/my-plan-body.jsx (hunk 1 above)
Comment (keoshi) [reply _inc/client/my-plan/my-plan-body.jsx]: How does this sound?

> High-speed, high-definition video hosting with no third-party ads.

We should also change the card in Calypso, where this copy originally came from:

image

https://github.com/Automattic/wp-calypso/blob/59bdfeeb97eda4266ad39410cb0a074d2c88dbc8/client/blocks/product-purchase-features-list/jetpack-video.jsx#L22

**Comment by keoshi** [comment]
@jeherve What browser are you using? Are you sure you were seeing the latest version of the styles on that page?

File: _inc/client/my-plan/my-plan-body.jsx (hunk 1 above)
Comment (jeherve) [reply _inc/client/my-plan/my-plan-body.jsx]: That works for me!

**Comment by jeherve** [comment]
> What browser are you using? Are you sure you were seeing the latest version of the styles on that page?

I was on Chrome, using a JN site so it should have been the latest version.

I'm happy to give it another try though! It seems you'll need to rebase the PR before we can merge it though.

File: _inc/client/my-plan/my-plan-body.jsx (hunk 1 above)
Comment (keoshi) [reply _inc/client/my-plan/my-plan-body.jsx]: Addressed in https://github.com/Automattic/wp-calypso/commit/17eaa1ae5eeba9343fc01178c5f04b5dce8a0e30

Also created a PR for Calypso in https://github.com/Automattic/wp-calypso/pull/29428

**Comment by keoshi** [comment]
@jeherve I had to change `nth-last-child` to `nth-last-of-type` because of the button container, which was affecting the count. Should be good to retest now.

**Comment by jeherve** [comment]
it seems there are still some layout issues:

image

**Comment by keoshi** [comment]
@jeherve Can you take a screenshot of the rules being applied there, please?

**Comment by keoshi** [comment]
@jeherve Ignore the last message, I found out what the problem was and have a fix coming right up.

**Review by jeherve — APPROVED** [review]
This looks good now! 🚢

**Comment by jeffgolenski** [comment]
@keoshi @joanrho I did notice however, that when viewing "my plan" in wp-admin, it's not apparent which plan I actually have. 

In the screenshot above, I'm actually have Jetpack Free, but the first words on the page are "Jetpack Premium"

Also, there's really not too many actionable things on this page for a free customer - even to upgrade it seems.

**Comment by keoshi** [comment]
@jeffgolenski Yeah, we could tackle that in a separate PR.

**Comment by jeffgolenski** [comment]
Extension of this here: https://github.com/Automattic/jetpack/issues/10995

**Pull request merged**